utilities/test_suite/HOST/runAudioTests.py

```diff
@@ -226,21 +226,21 @@ def rpp_test_suite_parser_and_validator():
         exit(0)
 
     for case in caseList:
-        if int(case) < 0 or int(case) > 1:
-            print(f"Invalid case number {case}. Case number must be 0-1 range!")
+        if int(case) < 0 or int(case) > 2:
+            print(f"Invalid case number {case}. Case number must be 0-2 range!")
             continue
 
         run_unit_test(srcPath, case, numRuns, testType, batchSize, outFilePath)
 else:
     for case in caseList:
-        if int(case) < 0 or int(case) > 1:
-            print(f"Invalid case number {case}. Case number must be 0-1 range!")
+        if int(case) < 0 or int(case) > 2:
+            print(f"Invalid case number {case}. Case number must be 0-2 range!")
```

Comment:
@sampath1117 Could you add a small change to have CASE_START and CASE_END as variables defined in the beginning so that we won't need to change start/end cases in lines 122,123,136,137,145,169,229,230,236,237.
One variable should control all this since this is prone to bugs. Pls introduce that as a small PR into non_silent_region itself.